src/components/swiper/swiper.vue

```diff
@@ -25,7 +27,10 @@
 </template>
 
 <script>
-    import Swiper from 'swiper';
+    import Swiper, { Pagination, Lazy } from 'swiper'
+    import SwiperSlides from './swiper-slides'
```

Comment:
@GianlucaGuarini When running tests I get following error: 

ERROR in ./src/components/swiper/swiper.vue?vue&type=script&lang=js& (./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/swiper/swiper.vue?vue&type=script&lang=js&)
Module not found: Error: Can't resolve './swiper-slides' in '/Users/jessicachrist/Documents/Repositories/vue-ui/src/components/swiper'
 @ ./src/components/swiper/swiper.vue?vue&type=script&lang=js& (./node_modules/babel-loader/lib!./node_modules/vue-loader/lib??vue-loader-options!./src/components/swiper/swiper.vue?vue&type=script&lang=js&) 36:0-43 53:18-30
 @ ./src/components/swiper/swiper.vue?vue&type=script&lang=js&
 @ ./src/components/swiper/swiper.vue
 @ ./src/components/swiper/swiper.spec.js

Unfortunately I haven't been able to figure out a solution yet

Reply:
@jessicachrist I was able to fix the problem, the issue was that here you need to explicitly define the file extension for your import BAD `import MyComponent from 'my-component'`  GOOD `import MyComponent from 'my-component.vue'`

Side note: these rules are not needed in the dreiup-template stack because [we force webpack loading also `.vue` files](https://github.com/dreipol/dreiup-templates/blob/ded6f6015fc1c45448cf0fc8ba122e790b47b683/_vue/gulp/tasks/compile-js/webpack-framework.config.js#L64) by default